File: BaekJoon/Array/8958.py
import sys

count = int(input())

for _ in range(count):
    s = sys.stdin.readline()
    result = 0
    score = 1
    for c in s:
        if c == "O":
            result+= score
            score+=1
        elif c == "X":
            score = 1

    print(result)

File: BaekJoonSwift/BaekJoonSwift/Array/8958.swift
import Foundation

let n = Int(readLine()!)!
var result = 0
var score = 0
for _ in 0..<n {
    let arr = Array(readLine()!)
    result = 0
    score = 0
    for c in arr {
        if c == "O" {
            score += 1
            result += score
        } else if c == "X" {
            score = 0
        }
    }
    
    print(result)
}
